### DISCUSSION TITLE
Fix long mod combinations overflowing from their boxes

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#19284] [Ideas] [answered]
Any mod combo longer than 7 mods in song select leaderboards and longer than 13 mods in the map "details" tab overflows
![osu_2022-07-21_19-43-47](https://user-images.githubusercontent.com/74463310/180279859-3c25beb5-b757-4ca6-95a4-d07003a1b771.jpg)
![osu_2022-07-21_15-36-01](https://user-images.githubusercontent.com/74463310/180279406-bcef5c0b-6bfb-417c-af43-f156fdea88b5.jpg)
 searched by "fit" , "overflow" "space" and came up empty so i dont think this has been brought up yet

**Comment by Joehuu** [comment, marked as answer]
It's a problem in multiple locations where mods are displayed. Already tracked in https://github.com/ppy/osu/issues/13952 / https://github.com/ppy/osu/issues/7923.

**Reply by mk56-spn** [reply]
id say it was worth bringing up again eitherway, the 2020 case was with a shorter mod combination and was seemingly addressed previously, this is related to the leaderboards which have only been an issue since they were switched to lazer scores
